File: projet/src/main/java/com/enicar/projet/dtos/LettreRequestDTO.java
package com.enicar.projet.dtos;


import lombok.Data;

@Data
public class LettreRequestDTO {

    // ── Étudiant ──────────────────────────────────────────────────────
    private String nomEtudiant;


    private String prenomEtudiant;


    private String cin;

    private String dateDelivranceCin;   // date de délivrance CIN  (dd/MM/yyyy)

    private String lieuDelivranceCin;   // lieu de délivrance CIN

    private String niveau;              // ex : "1", "2", "3"

    private String specialite;

    // ── Stage ─────────────────────────────────────────────────────────
    private String entreprise;

    private String dateDebut;           // format dd/MM/yyyy

    private String dateFin;             // format dd/MM/yyyy
}

File: front_spring/src/app/models/lettreRequestDTO.ts

export interface LettreRequestDTO {
  // Étudiant
  nomEtudiant: string;
  prenomEtudiant: string;
  cin: string;
  dateDelivranceCin: string; // format dd/MM/yyyy
  lieuDelivranceCin: string;
  niveau: string;            // ex: "1", "2", "3"
  specialite: string;

  // Stage
  entreprise: string;
  dateDebut: string;         // format dd/MM/yyyy
  dateFin: string;           // format dd/MM/yyyy
}